docs/application/tizen-studio/faq/faq.md

```diff
@@ -0,0 +1,180 @@
+         border-radius: 40px;
+         display: inline-block;
+         width: 47px;
+         height: 45px;
+         line-height: 28px;
+         font-size: 18px;
+         margin: 4px 1.5em 0px 0px;
+         color:#008aee;
+         }
+         .css-accordion {
+         list-style-type: none;
+         counter-reset: li;
+         padding: 20px;
+         margin: 20px;
+         width: 600px;
+         repeat top left;
+         /*---- End .accordion-item ----*/
+         }
+         .css-accordion .accordion-item {
+         counter-increment: li;
+         padding: 0;
+         margin: 0;
+         }
+         .css-accordion .accordion-item .item-content-container {
+         border-left: 9px solid #008aee;
+         padding: 7px 0px;
+         margin: -23px 0px -4px 25px;
+         }
+         .css-accordion .accordion-item .item-content-container .item-content {
+         background: #eee;
+         padding: 0px 0em;
+         overflow: hidden;
+         margin: 11px 0 0px 40px;
+         border-radius: 2px;
+         box-shadow: inset 0 2px 8px rgba(241, 235, 235, 0.5), 0 1px 2px rgba(255, 255, 255, 0.9);
+         }
+         .css-accordion .accordion-item .item-content-container .item-content p {
+         margin: 0.5em 0;
+         font-size: 14px
+         text-shadow: 0 1px 1px rgba(181, 99, 99, 0.9);
+         color: #090808;
+         }
+         .css-accordion .accordion-item input[type=radio] {
+         display: none;
+         /*---- End &:checked ----*/
+         }
+         .css-accordion .accordion-item input[type=radio] ~ .item-content-container {
+         overflow: hidden;
+         }
+         .css-accordion .accordion-item input[type=radio] ~ .item-content-container .item-content {
+         height: 0;
+         transition: all 0.3s linear;
+         }
+         .css-accordion .accordion-item input[type=radio]:checked ~ .item-content-container {
+         height: auto;
+         overflow: visible;
+         }
+         .css-accordion .accordion-item input[type=radio]:checked ~ .item-content-container .item-content {
+         height: inherit;
+         /*! overflow-y: auto; */
+         transition: all 0.5s linear;
+         }
+         .css-accordion .accordion-item input[type=radio]:checked + label {
+         color: #008aee;
+         }
+         .css-accordion .accordion-item input[type=radio]:checked + label:before {
+         background: #f3f3f3;
+         }
+      
+      </style>
+   </head>
+<body>
+   <ol class="css-accordion user-journey">
+      <li class="accordion-item stage-1">
+         <input type="radio" name="accordion-control" id="stage-1-control" checked="">
+            <label for="stage-1-control">How to run Tizen Studio on Ubuntu 18.04?</label>
+               <div class="item-content-container">
+                  <div class="item-content">
+                     <p>Tizen studio supports Ubuntu 18.04. However, Ubuntu 18.04 distribution does not include the runtime library <b>libpng12-0</b>.
```

Comment:
From which version of Tizen Studio Ubuntu 18.x is supported?

Rephrase to:
Tizen Studio is supported on Ubuntu 18.04. However, Ubuntu..